scraper/src/tests/conftest.py

```diff
@@ -0,0 +1,4 @@
+import os
```

Comment:
Ah yes make sense.
I think we can probably remove this file and change the command in the readme to:

```
API_KEY='test' APPLICATION_ID='test' ./docsearch test
```

What do you think ?